sklearn/ensemble/tests/test_stacking.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from sklearn.datasets import load_iris
+from sklearn.datasets import load_diabetes
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.linear_model import LinearRegression
+from sklearn.svm import LinearSVC
+from sklearn.svm import LinearSVR
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.ensemble import RandomForestRegressor
+
+from sklearn.ensemble import StackingClassifier
+from sklearn.ensemble import StackingRegressor
+
+from sklearn.model_selection import train_test_split
+from sklearn.model_selection import StratifiedKFold
+
+
+@pytest.mark.parametrize(
+    "cv",
+    [3, StratifiedKFold(shuffle=True, random_state=42)]
+)
+@pytest.mark.parametrize(
+    "meta_estimator",
+    [None, RandomForestClassifier(random_state=42)]
+)
+def test_stacking_classifier_iris(cv, meta_estimator):
+    X, y = load_iris(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
+    stacked_estimators = [('lr', LogisticRegression()),
+                          ('svc', LinearSVC())]
+    clf = StackingClassifier(estimators=stacked_estimators,
+                             meta_estimator=meta_estimator,
+                             cv=cv, random_state=42)
+    clf.fit(X_train, y_train)
+    clf.predict(X_test)
+    clf.predict_proba(X_test)
+    assert clf.score(X_test, y_test) > 0.8
+
+
+@pytest.mark.parametrize(
+    "cv",
+    [3, StratifiedKFold(shuffle=True, random_state=42)]
+)
+@pytest.mark.parametrize(
+    "meta_estimator",
+    [None, RandomForestRegressor(random_state=42)]
+)
+def test_stacking_regressor_diabetes(cv, meta_estimator):
+    X, y = load_diabetes(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
+    stacked_estimators = [('lr', LinearRegression()),
+                          ('svr', LinearSVR())]
+    clf = StackingRegressor(estimators=stacked_estimators,
+                            meta_estimator=meta_estimator,
```

Comment:
this should not be called meta_estimator. The StackingRegressor is the meta_estimator. This could be the ``base_estimator`` though that's a bit ambiguous in this context so maybe it's the ``top_`` or ``final_`` estimator?

No examples makes me less likely to review :P

Reply:
What about `estimators` replaced by `base_estimators` and `meta_estimator` by `stacking_estimator`

Reply:
just my two cents, but you can replace `estimators` with `level_0_estimators` and `meta_estimator` with `level_1_estimator` as used in the Stacked Generalization paper by Wolpert.

Can't wait for this feature!

Reply:
The naming conventions that proposed by @jmwoloso are right. But IMHO we should stick with the naming conventions of the framework. Currently, in `VotingClassifier`, which is another heterogeneous ensemble approach, we have the `estimators` parameter which is defined as follows:

>    estimators : list of (string, estimator) tuples
        Invoking the ``fit`` method on the ``VotingClassifier`` will fit clones
        of those original estimators that will be stored in the class attribute
        ``self.estimators_``. An estimator can be set to `None` using
        ``set_params``.

So, I believe that the parameter for the level 0 estimators should be named `estimators`. As for the level 1 estimator `stacker` or `final_estimator` could be an option.

Reply:
`final_estimator` is actually better than `stacker` because the stacking is done before that estimator :)

I basically agree with @chkoar and went with `estimators` and `final_estimator` :)